### DISCUSSION TITLE
Please add the ability to display current page number in the current section

### DISCUSSION THREAD

**Discussion by pitinata** [dolanmiu/docx#1622] [Ideas / Requests]
This is my first time I made a discussion here.

At the current version, page number in docxjs can display 3 different values.
1. Current page number (CURRENT)
2. Total page number (TOTAL_PAGES)
3. Total page number in the current section (TOTAL_PAGES_IN_SECTION)
But I would like to display current page number in the current section, which seems to be impossible to do.

I have already searched both in the documents and the docxjs codebase, but I still have no clue how to display it.
Is this features available?
If not, can I request this as a new features? I think having TOTAL_PAGES_IN_SECTION without CURRENT_PAGES_IN_SECTION is not make sense.

Thank you very much.

**Comment by pitinata** [comment]
I found a solution just after I posted this discussion.
I just add this properties before headers to change current page numbers to any number I want.

```
properties: {
                page: {
                    pageNumbers: {
                        start: 1,
                        formatType: NumberFormat.DECIMAL,
                    },
                },
            },
```

**Reply by dolanmiu** [reply]
I do not think `CURRENT_PAGES_IN_SECTION` is possible, and is not a feature of Microsoft Word. I have tried to do my research, and it does not seem even possible with Word
